person.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'nameable'
+class Person < Nameable
+  attr_accessor :id, :name, :age
+
+  def initialize(age, name = 'Unknown', _parent_permission = 1)
+    super()
+    @age = age
+    @name = name
+    @parent_permission = 1
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission
+  end
+
+  def correct_name
+    @name
+  end
+
```

Comment:
Great job @haftamudesta 🤝

- [ ] Kindly ensure a method for adding rentals is present. And the person itself should be sent as a parameter to create the new rental achieving the 'has-many' association. Kindly use this [link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more guidance.